### ISSUE TITLE
Input Group Accessibility issues

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#4667]
Input Group Accessibility issues

Attached the doc
[Input Group.docx](https://github.com/SAP/fundamental-ngx/files/5980075/Input.Group.docx)

**Comment by kavya-b** [comment]
There are around 6-7 issues which need analysis, therefore will take more time. Changed estimate accordingly. Will work on this from tomorrow.

**Comment by kavya-b** [comment]
- [ ] OB: On accessing the ‘Right Aligned Text Add-on”, the association label is not read 
EB: Ideally, all visually visible content should be conveyed to the screen reader user.
Note: The most of the field, the association label is not read.
- [ ] OB: No alternative text provided for the ‘Delete’ button, highlighted in red.
EB: Ideally, the alternative text must be provided for the user to identify the icon.
- [ ] OB: On adapting the screen to text spacing the highlighted text gets truncated.
EB: Ideally, there must be ability to increase spacing between lines, words, letters, and paragraphs, or any combination of these, may assist a user with effectively reading text. And the text must be adapted without loss of content or functionality.
**From Kavya: there are no visual specs for Input Group, therefore we don't think we can add extra lines to display longer content here. Also, the screen reader anyway reads the full text even if it is truncated, right? This issue will happen on any component that uses `fd-form-label` (Check _input, form message, textarea, step input, multi combobox/input_, etc.). However, for this particular case, we can provide the full text in the tooltip through the `title` attribute. Does this suffice?** @puru-hk @Sanjay-1312 
- [ ] OB: On accessing the Left Aligned Text Add-on is not conveyed to the user
EB: Ideally, all visual information must be conveyed to the user.
- [ ] OB: On accessing the button, additional information such as ‘Collapsed’ is conveyed.  (**2 issues of this type**)
EB: Ideally, such additional information must be removed, that may confuse the user.
- [ ] OB: On accessing the Playground, there is no keyboard support provided for the ‘Add’ button.
EB: Ideally, there should be keyboard support should be provided for the user who are highly dependent on the Keyboard should work alone on Keyboard
![image](https://user-images.githubusercontent.com/53509521/120422264-8424c100-c385-11eb-91c5-751e13e68cf9.png)
**From Kavya: The image shared in the document is of the Add icon, and is not a button. The button checkbox is not selected in the playground. Could you verify this with selecting the `button` checkbox again, @Sanjay-1312 @puru-hk ?**
